package pt.isec.pa.apoio_poe.model.Exceptions;

import pt.isec.pa.apoio_poe.model.data.Aluno;

import java.util.ArrayList;
import java.util.Arrays;
import java.util.HashSet;
import java.util.List;

public class ConflitoAtribuicaoAutomaticaException extends RuntimeException{

    public ConflitoAtribuicaoAutomaticaException() {
    }
}
